ott/geometry/costs.py

```diff
@@ -90,17 +90,88 @@ def tree_unflatten(cls, aux_data, children):
     return cls(*children)
 
 
+@jax.tree_util.register_pytree_node_class
+class RBFCost(CostFn):
+  """A radial-basis function cost class for translation invariant costs.
+
+  Such costs are defined as
+
+  c(x,y) = h(z), where z := x-y.
+
+  where h is a function strictly convex (or concave) function mapping vectors
```

Comment:
Just a small repetition here: I think you meant "where h is a strictly convex (or concave) function ...". It's minor, I know ;)

Reply:
great catch! thanks.